### PR TITLE
Feature/three models

### PR DESCRIPTION
**Pull request by hothanhtuan1209**
In this pull request, I create three models in three apps, run makemigration command to create database. In addition, I use django shell to test model
#84

### CODE REVIEW

File: django/manager_employees/contacts/models.py [hunk 1]
```diff
@@ -1,3 +1,29 @@
+import uuid
+
 from django.db import models
+from django.core.validators import RegexValidator
+
+from employees.models import Employee
+
+
+class Contact(models.Model):
```
Comment (cuongdoan-agilityio) [inline django/manager_employees/contacts/models.py:9]: Please help to update the docstring.

File: django/manager_employees/employees/models.py [hunk 2]
```diff
@@ -1,3 +1,64 @@
...
+
+    Attributes:
+        id (UUIDField): The primary key for the employee.
+        first_name (CharField): The first name of the employee.
+        last_name (CharField): The last name of the employee.
+        gender (CharField): The gender of the employee.
+        birthday (DateField): The employee's birth date.
+        email(CharField): The employee's email.
+        status (CharField): The status of the employee in company.
+        department.id (ForeignKey): This is the department in which
```
Comment (cuongdoan-agilityio) [inline django/manager_employees/employees/models.py]: I think it should be department instance, not department id

File: django/manager_employees/employees/models.py [hunk 3]
```diff
@@ -1,3 +1,64 @@
...
+
+        return f"{self.first_name} {self.last_name}"
+
+    def __str__(self):
+        """
+        Return a string representation of the Employee object.
+        """
 
-# Create your models here.
+        return str(self.get_full_name)
```
Comment (cuongdoan-agilityio) [inline django/manager_employees/employees/models.py]: Please help check this code.

File: django/manager_employees/employees/models.py (hunk 2 above)
Comment (hothanhtuan1209) [reply django/manager_employees/employees/models.py]: Done

File: django/manager_employees/employees/models.py (hunk 3 above)
Comment (hothanhtuan1209) [reply django/manager_employees/employees/models.py]: Done

**Pull request merged**